app/assets/stylesheets/new_styles/_base.scss

```diff
@@ -40,6 +42,15 @@ a { color : $link-blue  }
     height: 35px;
     width: 35px;
   }
+
+  &.medium {
+    height: auto;
+    width: auto;
+    max-width: 75px;
+  }
+
+  &.large {
+  }
```

Comment:
Yes, I misunderstood. I think it's a visual design problem of Github. Above the comment saying "this" are several lines, so I don't know what are you referring to.

Work around is to avoid saying "this" and say "line 52 and 53" or "two lines above this comment".

Root solution is to "paint" lines and comment. Then it refers to those painted lines and only those shown.